.pre-commit-config.yaml

```diff
@@ -19,6 +19,7 @@ repos:
   - id: end-of-file-fixer
   - id: mixed-line-ending
   - id: trailing-whitespace
+    args: [--markdown-linebreak-ext=md]
```

Comment:
Did you actually use a hard linebreak somewhere? I cannot tell from the diff.

Reply:
Yes, line 9 ` Its that time again!  ` idk it looks of when there is no linebreak after that.

Reply:
> Yes, line 9 ` Its that time again!  `  idk it looks off when there is no linebreak after that.

Ok